integrated-developer-portal/integrated-developer-portal.yaml

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2022 Google LLC
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+openapi: 3.0.0
+info:
+  title: Sample Integrated Developer Portal
+  description: A portal for an API key protected proxy
+  version: "1"
+servers:
+  - url: https://[YOUR_DOMAIN]/v1/samples/integrated-developer-portal
```

Comment:
why not have the script update this file since you are already sourcing the APIGEE_HOST variable

Reply:
I hadn't thought to do so. I'll work on it.